src/model/FavoriteProductsModel.ts

```diff
@@ -0,0 +1,66 @@
+import { Product } from "@/services/ProductAPI";
+import { getPostgresClient } from "@/services/server/database";
+async function getFavorites(userId: number) {
```

Comment:
Does this `getFavorites` work correctly, meaning: does it correctly return the props of the Product?